static/js/publisher/metrics/graphs/activeDevices.js

```diff
@@ -1,4 +1,6 @@
-/* globals bb, moment */
+/* globals bb */
```

Comment:
Can't `bb` be imported as well?

Reply:
unfortunately not! That's one of the reasons I want to rewrite it!